docs/content/en/docs/operator-manual/piped/configuring-notifications.md

```diff
@@ -16,22 +16,22 @@ Notification configuration including:
 [Notification Route](/docs/operator-manual/piped/configuration-reference/#notificationroute) matches events based on their metadata like `name`, `group`, `env`, `app`.
 Below is the list of supporting event names and their groups.
 
-| Event | Group |
-|-|-|
-| DEPLOYMENT_TRIGGERED | DEPLOYMENT |
-| DEPLOYMENT_PLANNED | DEPLOYMENT |
-| DEPLOYMENT_APPROVED | DEPLOYMENT |
-| DEPLOYMENT_WAIT_APPROVAL | DEPLOYMENT |
-| DEPLOYMENT_ROLLING_BACK | DEPLOYMENT |
-| DEPLOYMENT_SUCCEEDED | DEPLOYMENT |
-| DEPLOYMENT_FAILED | DEPLOYMENT |
-| DEPLOYMENT_CANCELLED | DEPLOYMENT |
-| APPLICATION_SYNCED | APPLICATION_SYNC |
-| APPLICATION_OUT_OF_SYNC | APPLICATION_SYNC |
-| APPLICATION_HEALTHY | APPLICATION_HEALTH |
-| APPLICATION_UNHEALTHY | APPLICATION_HEALTH |
-| PIPED_STARTED | PIPED |
-| PIPED_STOPPED | PIPED |
+| Event | Group | Supported |
+|-|-|-|
+| DEPLOYMENT_TRIGGERED | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| DEPLOYMENT_PLANNED | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| DEPLOYMENT_APPROVED | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| DEPLOYMENT_WAIT_APPROVAL | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| DEPLOYMENT_ROLLING_BACK | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| DEPLOYMENT_SUCCEEDED | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| DEPLOYMENT_FAILED | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| DEPLOYMENT_CANCELLED | DEPLOYMENT | <p style="text-align: center;"><input type="checkbox" checked></p> |
+| APPLICATION_SYNCED | APPLICATION_SYNC | <p style="text-align: center;"><input type="checkbox"></p> |
+| APPLICATION_OUT_OF_SYNC | APPLICATION_SYNC | <p style="text-align: center;"><input type="checkbox"></p> |
+| APPLICATION_HEALTHY | APPLICATION_HEALTH | <p style="text-align: center;"><input type="checkbox"></p> |
+| APPLICATION_UNHEALTHY | APPLICATION_HEALTH | <p style="text-align: center;"><input type="checkbox"></p> |
+| PIPED_STARTED | PIPED | <p style="text-align: center;"><input type="checkbox"></p> |
+| PIPED_STOPPED | PIPED | <p style="text-align: center;"><input type="checkbox"></p> |
```

Comment:
should add `disabled` property to those `input` tags or other will be above to click check/uncheck there 😄